docs/faq/stream-to.md

```diff
@@ -2,9 +2,9 @@
 
 Install the [squeezelite addon](https://github.com/pssc/ha-addon-squeezelite) which will then allow streaming over an audio connection from the HA host to your speaker or amplifier
 
-## My random connected device
+## My Random Connected Device
```

Comment:
Now I'm not a native English speaker but should this be capitalized ?
Or say it the other way: Can you explain me why you change it to capitals ?